build.sbt

```diff
@@ -0,0 +1,69 @@
+import play.core.PlayVersion
+import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+
+lazy val compileDeps = Seq(
+  "uk.gov.hmrc" %% "http-verbs" % "6.4.0",
+  "uk.gov.hmrc" %% "play-auditing" % "2.10.0",
+  "uk.gov.hmrc" %% "play-auth" % "1.1.0",
+  "uk.gov.hmrc" %% "play-config" % "4.3.0",
+  "uk.gov.hmrc" %% "play-graphite" % "3.2.0",
+  "uk.gov.hmrc" %% "play-health" % "2.1.0",
+  "uk.gov.hmrc" %% "logback-json-logger" % "3.1.0",
+  "de.threedimensions" %% "metrics-play" % "2.5.13",
+  "uk.gov.hmrc" %% "play-reactivemongo" % "5.2.0",
+  "uk.gov.hmrc" %% "microservice-bootstrap" % "5.16.0",
+  "org.typelevel" %% "cats" % "0.9.0"
```

Comment:
As first major cats version is in the pipeline maybe we should try to catch api changes early? [cats 1.0-MF](https://github.com/typelevel/cats/releases/tag/v1.0.0-MF)